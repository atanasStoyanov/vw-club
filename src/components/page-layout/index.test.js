import React from 'react';
import renderer from 'react-test-renderer';
import PageLayout from '.';
jest.mock('../header', () => 'Header');

describe('PageLayout Component', () => {
    it('should render page page layout', () => {
        const tree = renderer.create(<PageLayout />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})