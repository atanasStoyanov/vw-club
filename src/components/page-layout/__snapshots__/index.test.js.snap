// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout Component should render page page layout 1`] = `
<div
  className="app"
>
  <Header />
  <div
    className="container"
  >
    <div
      className="inner-container"
    />
  </div>
  <footer>
    <div
      className="footer"
    >
      <p
        className="copy-right"
      >
        Copyright Â© VW Passat Club 2020
      </p>
    </div>
  </footer>
</div>
`;
